commit_data.py

```diff
@@ -11,60 +11,46 @@
+              location="Kenya", votes=0, party="ODM"),
+    
+]
+
+deputy-presidents = [
     President(first_name="William", last_name="Ruto",
-              location="Kenya", votes=0),
+              location="Kenya"),
     President(first_name="Kalonzo", last_name="Musyoka",
-              location="Kenya", votes=0,)
+              location="Kenya")
 ]
 
 governors = [
     Governor(first_name="Lee", last_name="Kinyanjui",
-             location="Nyeri", votes=0),
-    Governor(first_name="Samuel", last_name="Kamau",
-             location="Nairobi", votes=0),
-    Governor(first_name="Agnes", last_name="Joe", location="Kisumu", votes=0),
-    Governor(first_name="Andrew", last_name="Onyango",
-             location="Mombasa", votes=0),
-    Governor(first_name="Sandra", last_name="Kyalo",
-             location="Kwale", votes=0),
-    Governor(first_name="Nancy", last_name="Kamulu",
-             location="Nairobi", votes=0,),
-    Governor(first_name="Mark", last_name="Owiti",
-             location="Mombasa", votes=0,),
-    Governor(first_name="Amina", last_name="Kitui",
-             location="Nyeri", votes=0,),
-    Governor(first_name="Luca", last_name="Kitui", location="Kwale", votes=0,),
-    Governor(first_name="Felista", last_name="Toms",
-             location="Kisumu", votes=0,)
+             location="Nakuru", votes=0, party="Jubilee"),
+    Governor(first_name="Mike", last_name="Mbuvi",
+             location="Nairobi", votes=0, party="jubilee"),
+    Governor(first_name="Josphat", last_name="Nanok", location="Turkana", votes=0, party="ODM")
+   
 ]
 
 senators = [
-    Senator(first_name="Alice", last_name="Kipkoech",
-            location="Nyeri", votes=0),
-    Senator(first_name="Jane", last_name="Oyando",
-            location="Nairobi", votes=0),
-    Senator(first_name="Michael", last_name="Mwaura",
-            location="Kisumu", votes=0),
-    Senator(first_name="Andrew", last_name="Juma",
-            location="Mombasa", votes=0),
-    Senator(first_name="Sandra", last_name="Mwikali",
-            location="Kwale", votes=0),
-    Senator(first_name="Lisa", last_name="Ali", location="Nairobi", votes=0,),
-    Senator(first_name="Kipkurui", last_name="Mercy",
-            location="Mombasa", votes=0,),
-    Senator(first_name="Sandy", last_name="Maalim",
-            location="Nyeri", votes=0,),
-    Senator(first_name="Abdi", last_name="Hussein",
-            location="Kwale", votes=0,),
-    Senator(first_name="Erick", last_name="Boaz", location="Kisumu", votes=0,)
+    Senator(first_name="Margaret", last_name="Kamar",
+            location="Uasin-Gishu", votes=0,party="ODM"),
+    Senator(first_name="Kipchumba", last_name="Murkomen",
+            location="Elgeyo-Marakwet", votes=0,party="Jubilee"),
+    Senator(first_name="Samuel", last_name="Poghisio",
+            location="West-Pokot", votes=0,party="ODM")
+
+]
+
+parties =[
+        party(party_name="ODM",party_logo_path=" ")
```

Comment:
Always declare classes starting with uppercase. Also, remember to separate items in list with commas.